cmd/build-cross-compile/build-cross-compile.go

```diff
@@ -82,6 +83,7 @@ func main() {
 		"PATH=" + strings.Replace(os.Getenv("PATH"), `\`, `\\`, -1),
 		"GOPATH=" + strings.Replace(os.Getenv("GOPATH"), `\`, `\\`, -1),
 		"GOROOT=" + runtime.GOROOT(),
+		"GOCACHE=" + filepath.Join(directory, ".cache"),
```

Comment:
Is this (and the makefile change) relevant?

Reply:
required for go1.12 support.